Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -7,47 +7,79 @@ import SwiftUI
 class DisplayHymnViewModel: ObservableObject {
 
     typealias Title = String
-
-    @Published var favorited: Bool = false
     @Published var title: Title = ""
     var hymnLyricsViewModel: HymnLyricsViewModel
     private let identifier: HymnIdentifier
     private let repository: HymnsRepository
     private let mainQueue: DispatchQueue
+    @Published var favoritedStatus = false
+    var hymnType = ""
+    var hymnNumber = ""
```

Comment:
I don't think these two are necessary. We should just pass in the HymnIdentifier into FavoritedEntity and unpack it in there. That way we use HymnIdentifier as much as possible, up until the point we absolutely cannot use it anymore, then we unpack it.

Reply:
Addressed in commit e9e8a89